packages/app-env/.flagshipcoderc.json

```diff
@@ -0,0 +1,22 @@
+{
+  "options": [
+    {
+      "flags": "--app-env-initial <value>",
+      "description": "Specifies the initial environment to be used when the application starts (e.g., 'development', 'staging', or 'production').",
+      "required": true,
+      "example": "--app-env-initial development"
+    },
+    {
+      "flags": "--app-env-dir <value>",
```

Comment:
We technically could aggregate this with babel traversal and looking at imports etc.